source/includes/fact-5.0-non-transactional-config-reads.rst

```diff
@@ -0,0 +1,10 @@
+Starting in MongoDB 5.0, non-transaction reads are not allowed on
+:ref:`config.transactions<config.transactions>` 
+with the following read concerns and considerations:
+
+- :ref:`read concern "snapshot"<readconcern-snapshot>`
+- :ref:`read concern "majority"<readconcern-majority>` and the 
+  :ref:`afterClusterTime<afterClusterTime>` option is set
+- When using a `MongoDB Driver <https://docs.mongodb.com/drivers/?tck=docs_server>`_  
```

Comment:
You can do:

```
:driver:`MongoDB Driver </>`
```

This is defined in the snooty rst spec: https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml